aioamqp/tests/test_channel.py

```diff
@@ -29,6 +29,16 @@ def test_close(self):
         self.assertFalse(channel.is_open)
 
     @testing.coroutine
+    def test_server_initiated_close(self):
```

Comment:
We probably have to try to create a new channel using `Protocol.channel()` (which should be rewriten to have both real arguments instead of `**kwargs` and a propoer docstring) with the expected closed id, and try to publish message using this new channel ?